Package.swift

```diff
@@ -6,32 +6,26 @@ import PackageDescription
 let package = Package(
     name: "UIExtensions",
     platforms: [
-//        .macOS(SupportedPlatform.MacOSVersion.v10_15),
```

Comment:
By removing lottie, I guess we can add support to all platforms again, right?
Not that we need that right now, but just for the sake of compiling libraries that do support multiple platforms and depend on UIExtensions

Reply:
Yes, done!